doc/rucio-server.md

```diff
@@ -0,0 +1,590 @@
+get it up and running. This server is a WSGI application written in
+Python and has to run on a WSGI capable web server such as Apache. The
+server provides a REST API and interacts with a database. The actual
+tasks are then performed by intermediate-level daemons. These read the
+database and can interact with low level components including
+storage.
+
+This architecture gives big flexibility in choices of authentication,
+transfer protocols, and storage systems, and no special software has
+to run on the storage systems as long as they provide a supported
+protocol and authN/authZ mechanism. However, this can also make
+first-time setup a nontrivial task. The Rucio developers recommend all
+users to run the Docker containers they provide. To learn about setup
+and configuration however, it may still be preferrable to install the
+Rucio server and necessary daemons from source. Another reason is that
+the provided containers and instructions for use are all built on
+CentOS, which may be incompatible with more up to date host operating
+systems. In the following we describe how to deploy a Rucio
+server on an Ubuntu 20.04 LTS or 21.10 server system.
+
+# Prerequisites
+
+In order to install Rucio from source you must have a system with
+Python, and the usual setup and compilation tools (build-essential, etc). We assume this to be
+available. Installing in a Python virtual environment may be
+preferrable for compatibility of dependencies. In the test setup
+described here, we instead started a fresh LXD container and installed
+all packages system-wide. Both the host and the container run Ubuntu
+20.04 LTS.
+
+# Dependencies
+
+## Install a web server
+
+A web server with support for the Python Web Server Gateway Interface
+(WSGI) is necessary. We adapted the Apache configuration files from
+the Dockers to the Debian/Ubuntu standard, as described below. Several
+optional Apache modules must be installed and configured for Rucio to
+work. Thus, as root/sudo:
+    apt install apache2 libapache2-mod-wsgi-py3 gridsite 
+
+Then enable the required Apache modules by creating symlinks in
+mods-enabled. a2enmod as root does this for you:
+
+	$sudo a2enmod wsgi
+
+Repeat this step for other required modules: 
+
+    authn, cache_disk, headers, proxy, rewrite, ssl, zgridsite
+
+(CHECKME which required modules are not installed and/or not enabled by default?)
+
+
+## Install required Python packages
+
+The Rucio distribution comes with a dependencies.txt file and setup
+will try to install the listed dependencies automatically as
+usual. Some of the packages may already be included in a standard
+Ubuntu install, but however some packages on PyPI or in the Ubuntu
+package archives are incompatible. These have to be installed or
+replaced separately.
+
+* **itsdangerous** is too new in PyPI but a compatible package exists in Ubuntu 20.04 
+     
+    $sudo apt install python3-itsdangerous
```

Comment:
I think you can remove that one. From 1.28 onwards it's not needed anymore. This was just due to an incompatibility between the flask version used and itsdangerous.